Core/src/org/sleuthkit/autopsy/modules/embeddedfileextractor/Bundle.properties-MERGED

```diff
@@ -1,5 +1,6 @@
 CannotCreateOutputFolder=Unable to create output folder.
 CannotRunFileTypeDetection=Unable to run file type detection.
+EmbeddedFileExtractor_make_output_dir_err=Failed to create module output directory for Embedded File Extractor
```

Comment:
This message is for an ingest module start up exception that is display to the user, so it must be localized.